application-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/config/__pluginName__ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package ${package}.config;
```

Comment:
I think this should be of the following form - if you compare with other plugins, like event-listener:

package ${package}.${pluginName};